src/components/Navigation/components/Item/tests/Item.test.tsx

```diff
@@ -468,8 +468,13 @@ function itemForLocation(location: string, overrides: Partial<ItemProps> = {}) {
       ]}
       {...overrides}
     />,
-    {
-      context: {location},
-    },
+    {location},
   ).find(Item);
 }
+
+function mountWithNavigationProvider(
```

Comment:
These mount* functions are only temporary and will be replaced once we're fully moved over to the new context API